src/Text.js

```diff
@@ -21,4 +21,7 @@ Text.propTypes = {
   ...textAlign.propTypes
 };
 
+Text.p = Text.withComponent('p');
```

Comment:
This might be a bit redundant, since Text is already a styled `p`

Reply:
Yeah, I was debating this as well. But figured it might be nice for consistency's sake to have the option to use `Text.p` if someone wanted.